apps/opencs/view/render/pagedworldspacewidget.hpp

```diff
@@ -124,6 +124,16 @@ namespace CSVRender
 
             virtual Cell* getCell(const osg::Vec3d& point) const;
 
+            virtual Cell* getCell(const CSMWorld::CellCoordinates& coords) const;
+
+            void setCellAlteredHeight(const CSMWorld::CellCoordinates& coords, int inCellX, int inCellY, float height);
+
+            float* getCellAlteredHeights(const CSMWorld::CellCoordinates& coords);
```

Comment:
Is this method used anywhere?

Reply:
Not any more. It was used during development, and apparently I forgot it here.